src/TLuaInterpreter.cpp

```diff
@@ -4388,12 +4388,12 @@ int TLuaInterpreter::getRoomExits( lua_State *L )
 // TODO: Provide exit details:
 int TLuaInterpreter::getAllRoomEntrances( lua_State *L )
 {
-    int roomId;
+    int roomId = 0;
     if( ! lua_isnumber( L, 1 ) ) {
         lua_pushstring( L, tr( "getAllRoomEntrances: bad argument #1 type (room id as number expected, got %1!)" )
                         .arg( luaL_typename( L, 1 ) )
                         .toUtf8().constData());
-        lua_error( L );
+        return lua_error( L );
```

Comment:
Have a look at the commit message - https://github.com/ahmedcharles/Mudlet/commit/c4cac9de8316a636b2e9adbf9e729391d07e4606 - it's just to silence a warning :)